src/plugins/fs/mount.c

```diff
@@ -140,6 +140,7 @@ static gboolean do_unmount (MountArgs *args, GError **error) {
     int ret = 0;
     gboolean success = FALSE;
 
+    mnt_init_debug (0);
```

Comment:
Mounting is only a small part of the FS plugin, but I think moving this to `bd_fs_init` won't hurt.

Reply:
Good idea, this will actually setup debugging early on udisksd startup, spitting debug output even for parsing fstab, mountinfo etc.